Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleSceneView.swift

```diff
@@ -0,0 +1,262 @@
+                    clippingDistance: clippingDistance,
+                    initialCameraIsSet: $initialCameraIsSet,
+                    isCalibrating: isCalibrating,
+                    locationDataSource: locationDataSource,
+                    sceneView: sceneViewBuilder
+                )
+            case .worldTracking:
+                WorldTrackingSceneView(
+                    calibrationViewModel: calibrationViewModel,
+                    clippingDistance: clippingDistance,
+                    initialCameraIsSet: $initialCameraIsSet,
+                    isCalibrating: isCalibrating,
+                    locationDataSource: locationDataSource,
+                    sceneView: sceneViewBuilder
+                )
+            }
+        }
+        .onDisappear {
+            Task { await locationDataSource.stop() }
+        }
+        .task {
+            // Start the location data source when the view appears.
+            
+            // Request when-in-use location authorization.
+            // The view utilizes a location datasource and it will not start until authorized.
+            let locationManager = CLLocationManager()
+            if locationManager.authorizationStatus == .notDetermined {
+                locationManager.requestWhenInUseAuthorization()
+            }
+            if !checkTrackingCapabilities(locationManager) {
+                print("Device doesn't support full accuracy location.")
+            }
+            
+            do {
+                try await locationDataSource.start()
+            } catch {
+                self.error = error
+            }
+            for await location in locationDataSource.locations {
+                currentLocation = location
+            }
+        }
+        .task {
+            do {
+                geoTrackingIsAvailable = try await checkGeoTrackingAvailability()
+            } catch {
+                self.error = error
+            }
+        }
+        .overlay(alignment: calibrationViewAlignment) {
+            if !calibrationViewIsHidden {
+                if !isCalibrating {
+                    Button {
+                        withAnimation {
+                            isCalibrating = true
+                        }
+                    } label: {
+                        Text("Calibrate")
+                            .padding()
+                    }
+                    .background(.regularMaterial)
+                    .clipShape(RoundedRectangle(cornerRadius: 10))
+                    .disabled(!initialCameraIsSet)
+                    .padding()
+                    .padding(.vertical)
+                }
+            }
+        }
+        .overlay(alignment: .bottom) {
+            if isCalibrating {
+                CalibrationView(viewModel: calibrationViewModel, isPresented: $isCalibrating)
+                    .padding(.bottom)
+            }
+        }
+        .overlay(alignment: .top) {
+            accuracyView
```

Comment:
Depending on what we want to display in the accuracy view, a few "onSomethingChanged" modifiers may be needed in the child views. They can be done in a separate PR.